.env

```diff
@@ -0,0 +1 @@
+VITE_APPWRITE_TWITTER_CLONE_PROJECT_ID=672daeb5001aed5a1c1b
```

Comment:
the env file should not be pushed to github ever

Reply:
env file added to gitIgnore folder